config/Makefile.am

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (C) 2024 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# See file LICENSE for terms.
+#
+
+conf_files_dir = $(ucx_config_dir)
+conf_files__DATA = ucx.conf
```

Comment:
i think need single underscore, not a double

Reply:
With single it will be:
```
conf_filesdir = $(ucx_config_dir)
conf_files_DATA = ucx.conf
```

Reply:
IMO it is better